src/m1_facilitator/models/Message.ts

```diff
@@ -126,4 +126,14 @@ export default class Message extends Comparable<Message> {
 
     return 0;
   }
+
+  /**
+   * Check if message type is withdraw or not.
+   */
+  public isWithdrawType(): boolean {
+    if (this.type === MessageType.Withdraw) {
```

Comment:
May be just:
`return this.type === MessageType.Withdraw;`

Reply:
Removed the method, it's not useful.